translations/plurals.ts

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!DOCTYPE TS>
-<TS version="2.0" language="en">
```

Comment:
I can do this. Just be sure to notice that my initial translation had `version="2.1"` too. It's probably related to the version of QtLinguist.

Isn't it better to use the newer version of the spec?